serverless.yml

```
@@ -4,6 +4,11 @@ service: animl-api
 
 plugins:
   - serverless-offline
+  - serverless-plugin-typescript
```

FYI: you might've already seen [this](https://github.com/serverless/serverless-plugin-typescript?tab=readme-ov-file#usage-with-serverless-offline), but switching up load order for `serverless-offline` might prevent future headache

```suggestion
  - serverless-plugin-typescript
  - serverless-offline
```